tests/test_a11y.py

```diff
@@ -112,3 +112,18 @@ def test_axe_core_kitchen_sink(
 
     # Expect Axe-core to have found 0 accessibility violations
     assert len(results["violations"]) == 0, pretty_axe_results(results)
+
+
+def test_version_switcher_highlighting(page: Page, url_base: str) -> None:
+    """This isn't an a11y test, but needs a served site for Javascript to inject the version menu."""
```

Comment:
@trallard In a follow-up PR I'd like to add a `@requires_playwright` decorator so that I can move this test out of `test_a11y.py` into `test_build.py`.  Something similar to 

https://github.com/mne-tools/mne-python/blob/68e4f89763135637110859ee0e05e594955815f1/mne/utils/_testing.py#L67-L81

OK with you if this non-a11y test squats in the `test_a11y.py` file for now until that gets done?  (did it this way because of the `importorskip` issue; didn't want to `importorskip` the entire other tests file when only one test needed playwright)

Reply:
sure, this makes sense to me